background.js

```diff
@@ -3,58 +3,61 @@ chrome.alarms.create('read later notifier', { delayInMinutes : 1, periodInMinute
 
 // set up addListener for notifies read later contents periodically
 chrome.alarms.onAlarm.addListener(function(alarm){
-    console.log('test: read-later-notification')
     
     // get all bookmark tree
     chrome.bookmarks.getTree(function(bookmark){
+        //TODO: enable to select directory name and content item number in setting menu
+        var selectedDirectoryName = "read_later"
+        var selectedItemNumber = 3
         var root = bookmark[0]['children']
         var bookMarkBar = root[0]['children']
-        var readLater = bookMarkBar[11]['children']
 
-        // get random elements from readLater 
-        var selectedReadLater = getRandomIntArray(readLater, 3)
-        
-        selectedReadLater.forEach(function(item, index, array) {
-            console.log(index, item)
-            console.log(item.id)
-            console.log(item.title)
-            console.log(item.url)
+        // get read_later directory from book mark bar
```

Comment:
`read_later` is not directory name, it's just the concept of reading later directory.

Reply:
Fix